packages/mwp-app-render/src/components/uxcapture/UXCaptureConfig.jsx

```diff
@@ -0,0 +1,37 @@
+// @flow
+import React from 'react';
+
+type callbackType = (label: string) => mixed;
+type Props = {|
+	onMark?: callbackType,
+	onMeasure?: callbackType,
+|};
+
+export default ({ onMark, onMeasure }: Props) => {
+	const config = {};
+
+	// No need to inject `UX.config()` script if no callbacks provided
+	if (!onMark && !onMeasure) {
+		return null;
+	}
+
+	if (onMark) {
+		config['onMark'] = onMark.toString();
+	}
+
+	if (onMeasure) {
```

Comment:
Any thoughts on maybe using the Uglify JS API here - https://github.com/mishoo/UglifyJS2#api-reference - to minify, strip comments - here? Might make the rendered html output a bit more concise.

Reply:
Given the size of the HTML snippet (< 0.5kB), I don't think I would make the effort.